ignite/services/plugin/template/integration/integration_test.go.plush

```diff
@@ -0,0 +1,69 @@
+package integration_test
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	pluginsconfig "github.com/ignite/cli/v28/ignite/config/plugins"
+	"github.com/ignite/cli/v28/ignite/pkg/cmdrunner/step"
+	"github.com/ignite/cli/v28/ignite/services/plugin"
+	envtest "github.com/ignite/cli/v28/integration"
+)
+
+func Test<%= Title %>(t *testing.T) {
+	var (
+		require = require.New(t)
+		env     = envtest.New(t)
+		app     = env.Scaffold("github.com/test/test")
+
+		assertPlugins = func(expectedLocalPlugins, expectedGlobalPlugins []pluginsconfig.Plugin) {
+			localCfg, err := pluginsconfig.ParseDir(app.SourcePath())
```

Comment:
It would be good to mark it as a helper:

```suggestion
		assertPlugins = func(t *testing.T, expectedLocalPlugins, expectedGlobalPlugins []pluginsconfig.Plugin) {
			t.Helper()

			localCfg, err := pluginsconfig.ParseDir(app.SourcePath())
```

It might be useful to split `assertPlugins` into `assertLocalPlugins` and `assertGlobalPlugins`, both could potentially be functions defined outside the test so they can be used within other tests and also to have a leaner test function.